src/components/TodoComponents/Todo.js

```diff
@@ -3,8 +3,9 @@ import './Todo.css';
 
 const Todo = props => {
   return (
-    <p className="list-text">{props.toDoItem.task}</p>
+    <p className={`list-text ${props.toDoItem.completed}`} onClick={() => props.changeStyle(props.toDoItem.id)} >{props.toDoItem.task}</p>
```

Comment:
This looks like it ought to be broken up for readability:
```javascript
<p
  className={`list-text ${props.toDoItem.completed}`}
  onClick={() => props.changeStyle(props.toDoItem.id)} 
>
  {props.toDoItem.task}
</p>
```
I'd also recommend some [object destructuring](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Object_destructuring) here:
```javascript
const Todo = ({ changeStyle, todoItem }) => {
  const { completed, id, task } = todoItem;
  return (
    <p
      className={`list-text ${completed}`}
      onClick={() => changeStyle(id)} 
    >
      {task}
    </p>
  );
};
```